composer.json

```diff
@@ -40,7 +40,8 @@
         "psr/log": "~1.0",
         "monolog/monolog": "~1.6",
         "pimple/pimple": "~1.1",
-        "robmorgan/phinx": "~0.4"
+        "robmorgan/phinx": "~0.4",
+        "krausv/ftp-deployment": "v2.1-RC"
```

Comment:
I wouldn't be comfortable merging this until we can use the mainline dg/ftp-deployment. Is there work underway to merge this fork back into the mainline?